diagnostic.md

```diff
@@ -49,7 +53,11 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
+@function tint($cornflower, 20%) {
+  @return mix(white, $cornflower, 20);
+}
+
+/* Not sure if you wanted the entire function coded or just the function call tint($cornflower, 20%); */
```

Comment:
Just the function call. Actually, your function isn't quite right, but I won't take points off for it.

Here's what it should look like:
```
$pure-white: #FFF;
@function tint($color, $percentage) {
  @return mix($pure-white, $color, $percentage);
}
```